httpd/apache.conf

```diff
@@ -0,0 +1,19 @@
+LoadModule wsgi_module modules/mod_wsgi.so
+LoadModule ssl_module modules/mod_ssl.so
+
+Listen 5001 https
+
+<VirtualHost *:5001>
+  SSLEngine On
+  SSLCertificateFile /PATH/TO/CERT.crt
+  SSLCertificateKeyFile /PATH/TO/KEY.key
+
+  WSGIPassAuthorization On
+  WSGIChunkedRequest On
+  WSGIDaemonProcess webtool user=apache group=apache threads=5 home=/var/www/setpass/
```

Comment:
That bit is copy-paste from a k2k script that I didn't change, so certainly open to discussion/critique.  I'll read up on the options when I get a chance.